Language/Classes/Instance_Methods/Operators/optional_parameter_t07.dart

```diff
@@ -4,11 +4,11 @@
 
 /// @assertion It is a compile-time error to declare an optional parameter in an 
 /// operator.
+///
```

Comment:
In line 21, please make adjustments similar to the ones in optional_parameter_t05.